UXWContributors.js

```diff
@@ -1,13 +1,104 @@
+const data = [ 
+  {
+    name: 'Keri',
+    role: 'Product Designer',
+    email: 'example@example.com',
+    image: '/assets/keri.jpg'
+  },
+  {
+    name: 'Laura',
+    role: 'Co-Organizer of UX Wizards',
+    email: 'example@example.com',
+    image: '/assets/laura.jpg'
+  },
+  {
+    name: 'Seiko',
+    role: 'Product Designer',
+    email: 'example@example.com',
+    image: '/assets/seiko.jpg'
+  },
+  {
+    name: 'Kaisen',
+    role: 'Software',
+    email: 'example@example.com',
+    image: '/assets/kaisen.jpg'
+  },
+  {
+    name: 'Javi',
+    role: 'Frontend Developer',
+    email: 'example@example.com',
+    image: '/assets/javi.jpg'
+  },
+  {
+    name: 'Max',
+    role: 'Product Designer',
+    email: 'example@example.com',
+    image: '/assets/max.jpg'
+  },
+  {
+    name: 'Sidney',
+    role: 'Product Designer',
+    email: 'example@example.com',
+    image: '/assets/sidney.jpg'
+  },
+  {
+    name: 'Dhruv',
+    role: 'Software',
+    email: 'example@example.com',
+    image: '/assets/dhruv.jpg'
+  },
+]
+
+const buttonStyle = {
+  outline: "0",
+  display: "inline-block",
+  padding: "8px",
+  "font-size":"15px",
+  color: "white",
+  "background-color": "#6331FD",
+  "text-align": "center",
+  cursor: "pointer",
+  width: "100%"
+}
+
+const titleStyle = {
+  color: "grey"
+}
+
+const columnStyle = {
+  float: "left",
+  width: "33.3%",
+  "margin-bottom": "16px",
+  padding: "0 8px"
+}
+
+const cardStyle = {
+  "box-shadow": "0 4px 8px 0 rgba(0, 0, 0, 0.2)",
+  "font-size": "10px"

```

Comment:
**Suggestion**(non-blocking): Text is rather small IMO. I would default to [body text being one root em](https://www.w3.org/Style/Examples/007/units.en.html). Browsers default this value to be `16px`, so I would start here and make adjustments from this baseline